part0/sqd_ex4.md

```diff
@@ -0,0 +1,11 @@
+```mermaid
+sequenceDiagram
+    participant User
+    participant Page
+    User->>Page: Writes something into the text field
+    User->>Page: Clicks the Save button
+    Note over Page: Browser operation: Validate input
+    Note over Page: Server operation: Save note to database
+
```

Comment:
@Rajdeep-Chakraborty0611 This does not include the HTTP request flow information as depicted in https://fullstackopen.com/en/part0/fundamentals_of_web_apps#loading-a-page-containing-java-script-review

As the assignment states, you can use any other tool than using Mermaid syntax if it helps E.g. Google Draw to create the sequence diagram. However, it must show the required information of data flow, types of HTTP methods invoked, browser<->server interaction, etc.